Modix/StringULongConverter.cs

```diff
@@ -1,25 +0,0 @@
-﻿using System;
-using Newtonsoft.Json;
-
-namespace Modix
-{
-    public class StringULongConverter : JsonConverter
```

Comment:
Was this only used by the Rowboard importer? Or is it just obsolete? I seem to recall @jmazouri being the one who added this, or at least knowing why it was added. There was some particular issue that this was a workaround for.

Reply:
From what I can tell it wasn't used at all. Not even in the Rowboat importer

Reply:
This was for cases where ulong IDs were sent as numbers in json, and in clientside JavaScript, they were treated as floating point numbers and lost precision.

I'm not sure when it became unused, but if it's no longer an issue, this isn't necessary.

Reply:
This was for cases where ulong IDs were sent as numbers in json, and in clientside JavaScript, they were treated as floating point numbers and lost precision.

I'm not sure when it became unused, but if it's no longer an issue, this isn't necessary.